### PR TITLE
Miscellaneous updates to theme object

### PR DESCRIPTION
**Pull request by galacemiguel**
- Move `space` definition to theme.js
- Use `radii` key name to align with [System UI theme specification](https://system-ui.com/theme)
- Remove component-specific shadows

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -6,18 +6,9 @@ module.exports = {
     fontFamily: theme.fonts.body,
     fontSize: theme.fontSize,
     colors: theme.colors,
-    borderRadius: theme.borderRadius,
+    borderRadius: theme.radii,
```
Comment (eugiellimpin) [inline tailwind.config.js:9]: not sure how this is a better name 🤔

Comment (galacemiguel) [reply tailwind.config.js:9]: > Use `radii` key name to align with [System UI theme specification](https://system-ui.com/theme)

Just want to follow the System UI theme spec. 👍

**Comment by eugiellimpin** [comment]
LGTM

File: tailwind.config.js (hunk 1 above)
Comment (eugiellimpin) [reply tailwind.config.js:9]: okay!

**Comment by galacemiguel** [comment]
Thanks for the review! @eugiellimpin

**Pull request merged**